### ISSUE TITLE
create org-wide issue template

### ISSUE THREAD

**Issue by nickreich** [Infectious-Disease-Modeling-Hubs/.github#11]
Problem statement: we want all new issues created in any repo in this GitHub org to land in the [main project for the organization](https://github.com/orgs/Infectious-Disease-Modeling-Hubs/projects/3/views/1)

Possible solution:
Per [this comment](https://github.com/orgs/community/discussions/22451#discussioncomment-3236785) we are thinking that it would be good to have an issue template organization-wide that is standard and also funnels ALL newly created issues into the [main project for the organization](https://github.com/orgs/Infectious-Disease-Modeling-Hubs/projects/3/views/1). Note that [this comment](https://github.com/orgs/community/discussions/9687#discussioncomment-6703290) documents an issue template using YML that can specify a default project. 

We are not sure how this would interact with the existing issue templates. Noting that hubEvals does not have an issue template currently, but most other hubverse R packages do. Once this is solved and an org-wide issue template is merged, would be nice to check (and report back with issues, if undesired behavior is observed) how different repos use (or not) the org-wide template.

**Comment by annakrystalli** [comment]
Good idea! I believe if there is an issue template within individual repos with the same name it overrides the organisation one within that repo so probably best to delete them where they exist.

**Comment by nickreich** [comment]
I'm getting cold feet now that I saw [this](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax)
![image](https://github.com/Infectious-Disease-Modeling-Hubs/.github/assets/1280767/0326472b-bebf-40ec-a241-81fda2328c39)

But the thought of going through each individual repo and setting up the workflows also feels cumbersome... @bsweger for additional thoughts?

**Comment by bsweger** [comment]
Interesting...

I actually think it's a Good Thing™️ that people who aren't explicitly on the Hubverse "team" can't open up issues that automatically get added to our project. 

Would we feel comfortable enabling write access to the Hubverse development overview project for every member of the Infectious-Disease-Modeling org? 

That way:

* Members of the Hubverse org (i.e., the org that will soon be formerly known as IDM) have their issues added automatically because they have required context about the work
* External contributors who don't have the same context would need to have their issues explicitly triaged by a Hubverse member and added to the project manually

Obviously, this idea only works f external contributors can create issues without getting a "lack of write access" error message (_i.e._, they create issues and GitHub just silently skips the "moving to the board" step).
